logisland-components/logisland-processors/logisland-processor-web-analytics/src/test/java/com/hurence/logisland/processor/webAnalytics/URLDecoderTest.java

```diff
@@ -176,4 +194,24 @@ public void testNoMatchingField() {
         out.assertFieldEquals("url2", urlVal2);
         out.assertFieldEquals("val4", val4);
     }
+
```

Comment:
I guess a unit test with '%ab' and '%AB' would be a good idea ?

Reply:
The tests you suggest are already covered (see below)